jerry-ext/handler/handler-print.c

```diff
@@ -65,7 +65,7 @@ jerryx_handler_print (const jerry_value_t func_obj_val, /**< function object */
       jerry_length_t substr_pos = 0;
       jerry_char_t substr_buf[256];
 
-      while ((substr_size = jerry_substring_to_char_buffer (str_val,
+      while ((substr_size = jerry_substring_to_utf8_char_buffer (str_val,
                                                             substr_pos,
```

Comment:
wrond indentation

Reply:
Thanks, I have fixed the indentation and updated the PR.